client/src/styles/components/_profile.scss

```diff
@@ -49,4 +49,10 @@
 		overflow-x: auto;
 	}
 
+	button {
```

Comment:
There's already a button style defined in the component directory. Do you need to define a new style? If you do, please define the style there.